### PR TITLE
Manifest, Windows: Support web manifest "orientation" field

### PR DESCRIPTION
**Pull request by astojilj**
BUG=XWALK-4862

### CODE REVIEW

**Comment by astojilj** [comment]
@darktears , @r0b5t4 please review. Thanks.

**Comment by darktears** [comment]
@pozdnyakov any feedback?

**Comment by pozdnyakov** [comment]
lgtm

**Comment by astojilj** [comment]
Let's not integrate it yet. New patch coming. Just verified on Windows7 - SetDisplayAutoRotationPreferences requires runtime symbol resolution from user32.dll as not available on Windows 7.

File: runtime/browser/runtime_platform_util_win.cc [hunk 1]
```diff
@@ -212,4 +213,76 @@ bool IsVisible(gfx::NativeView view) {
...
+    GetCurrentDisplaySettings(&landscape, &flipped);
+    if (landscape) {
+      prefs = flipped ? ORIENTATION_PREFERENCE_LANDSCAPE_FLIPPED
+                      : ORIENTATION_PREFERENCE_LANDSCAPE;
+    } else {
+      prefs = flipped ? ORIENTATION_PREFERENCE_PORTRAIT_FLIPPED
+                      : ORIENTATION_PREFERENCE_PORTRAIT;
+    }
+  } else {
+    assert(false);
```
Comment (pozdnyakov) [inline runtime/browser/runtime_platform_util_win.cc]: NOTREACHED() << "explanation"

Comment (astojilj) [inline runtime/browser/runtime_platform_util_win.cc]: > NOTREACHED() << "explanation"
The patch series with astojilj/crosswalk@ 8862f4d ...
Done.

**Comment by astojilj** [comment]
from irc:
[14:19] <astojilj> https://github.com/crosswalk-project/chromium-crosswalk/pull/322 and https://github.com/crosswalk-project/crosswalk/pull/3588

[14:23] <MPozdnyakov> astojilj: both are good

**Comment by astojilj** [comment]
Different patchsets are also available here
https://chromiumcodereview.appspot.com/1758823004/

**Pull request merged**